core/src/main/java/org/apache/iceberg/BaseTableScan.java

```diff
@@ -318,7 +318,7 @@ private Schema lazyColumnProjection() {
     return schema;
   }
 
-  private static String formatTimestampMillis(long millis) {
+  protected static String formatTimestampMillis(long millis) {
```

Comment:
This isn't used. Could you revert this change?

Reply:
OK